overlay/files/system/oem/00_rootfs.yaml

```diff
@@ -13,6 +13,10 @@ stages:
         providers: ["aws", "gcp", "openstack", "cdrom"]
         path: "/oem"
   rootfs:
+    - name: "Mount BPF on Alpine systems"
```

Comment:
maybe "Mount BPF on OpenRC system" would be a better description here?

Reply:
I want to talk with @mudler about this, because I dont get why its there. AFAIK the bpf tools in the alpine ecosystem are non-existant and this is possible by using the opensuse/ubuntu kernels. 

Not sure if this will be there for a long time :D

But yeah, maybe openrc systems would be nicer.

Reply:
gotcha, not a srtong request from my side, if we can get rid of it even better I guess